aether-couchdb-sync-module/setup.py

```diff
@@ -16,14 +18,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import os
+from io import open
 from setuptools import find_packages, setup
 
-os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+def read(f):
+    return open(f, 'r', encoding='utf-8').read()
+
 
 setup(
+    version=read('/var/tmp/VERSION').strip(),
     name='aether.sync',
-    description='A python module with Aether CoudhDB Sync functionality',
+    description='A python library with Aether CoudhDB Sync functionality',
```

Comment:
This is actually still a module, isn't it?

Reply:
That's the point, from the python perspective it's a library from the aether ecosystem perspective it's a module :woman_shrugging: 

What should we do?

Reply:
`An aether module with CouchDB Sync functionality`?

Reply:
Yes, I think that's the best option (but `Aether`, not `aether`)